<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Results (powered by Planemo)"
           tests="44"
           errors="5"
           failures="4"
           skip="0">
    
    <testcase classname="bcftools_norm (Test #1)" name="0" time="37.99066352844238">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            Lines   total/split/realigned/skipped:21/0/13/0
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="bcftools_norm (Test #2)" name="1" time="">
        
            <error type="error" message=""><![CDATA[
                { "status": "skip", "test_index": 1, "tool_id": "bcftools_norm", "tool_version": "1.15.1+galaxy4"}
            ]]></error>
        
    </testcase>
    
    <testcase classname="bcftools_norm (Test #3)" name="2" time="30.823127508163452">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            Lines   total/split/realigned/skipped:19/7/0/0
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="bcftools_norm (Test #4)" name="3" time="30.110564947128296">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            Lines   total/split/realigned/skipped:19/7/19/0
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="bcftools_norm (Test #5)" name="4" time="18.920686721801758">
        
            <error type="error" message="Input staging problem: Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 1436, in verify_tool
    stage_data_in_history(
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 175, in stage_data_in_history
    staging_interface.handle_jobs()
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 212, in handle_jobs
    self.galaxy_interactor.wait_for_job(job_response["id"], job_response["history_id"], self.maxseconds)
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 399, in wait_for_job
    self.wait_for(lambda: self.__job_ready(job_id, history_id), maxseconds=maxseconds)
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 403, in wait_for
    wait_on(func, what, walltime_exceeded)
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/wait.py", line 38, in wait_on
    value = function()
            ^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 399, in <lambda>
    self.wait_for(lambda: self.__job_ready(job_id, history_id), maxseconds=maxseconds)
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 749, in __job_ready
    return self._state_ready(job_id, error_msg="Job in error state.")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 841, in _state_ready
    raise Exception(
Exception: Job in error state.. tool_id: __DATA_FETCH__, exit_code: None, stderr: .
"><![CDATA[
                { "execution_problem": "Input staging problem: Traceback (most recent call last):\n  File \"/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 1436, in verify_tool\n    stage_data_in_history(\n  File \"/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 175, in stage_data_in_history\n    staging_interface.handle_jobs()\n  File \"/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 212, in handle_jobs\n    self.galaxy_interactor.wait_for_job(job_response[\"id\"], job_response[\"history_id\"], self.maxseconds)\n  File \"/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 399, in wait_for_job\n    self.wait_for(lambda: self.__job_ready(job_id, history_id), maxseconds=maxseconds)\n  File \"/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 403, in wait_for\n    wait_on(func, what, walltime_exceeded)\n  File \"/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/wait.py\", line 38, in wait_on\n    value = function()\n            ^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 399, in \u003clambda\u003e\n    self.wait_for(lambda: self.__job_ready(job_id, history_id), maxseconds=maxseconds)\n                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 749, in __job_ready\n    return self._state_ready(job_id, error_msg=\"Job in error state.\")\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 841, in _state_ready\n    raise Exception(\nException: Job in error state.. tool_id: __DATA_FETCH__, exit_code: None, stderr: .\n", "status": "error", "test_index": 4, "time_seconds": 18.920686721801758, "tool_id": "bcftools_norm", "tool_version": "1.15.1+galaxy4"}
            ]]></error>
        
    </testcase>
    
    <testcase classname="bcftools_norm (Test #6)" name="5" time="29.431668281555176">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            Lines   total/split/realigned/skipped:26/0/0/0
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="bcftools_norm (Test #7)" name="6" time="10.644328117370605">
        
            <error type="error" message="Input staging problem: Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 1436, in verify_tool
    stage_data_in_history(
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 175, in stage_data_in_history
    staging_interface.handle_jobs()
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 212, in handle_jobs
    self.galaxy_interactor.wait_for_job(job_response["id"], job_response["history_id"], self.maxseconds)
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 399, in wait_for_job
    self.wait_for(lambda: self.__job_ready(job_id, history_id), maxseconds=maxseconds)
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 403, in wait_for
    wait_on(func, what, walltime_exceeded)
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/wait.py", line 38, in wait_on
    value = function()
            ^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 399, in <lambda>
    self.wait_for(lambda: self.__job_ready(job_id, history_id), maxseconds=maxseconds)
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 749, in __job_ready
    return self._state_ready(job_id, error_msg="Job in error state.")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 841, in _state_ready
    raise Exception(
Exception: Job in error state.. tool_id: __DATA_FETCH__, exit_code: None, stderr: .
"><![CDATA[
                { "execution_problem": "Input staging problem: Traceback (most recent call last):\n  File \"/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 1436, in verify_tool\n    stage_data_in_history(\n  File \"/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 175, in stage_data_in_history\n    staging_interface.handle_jobs()\n  File \"/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 212, in handle_jobs\n    self.galaxy_interactor.wait_for_job(job_response[\"id\"], job_response[\"history_id\"], self.maxseconds)\n  File \"/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 399, in wait_for_job\n    self.wait_for(lambda: self.__job_ready(job_id, history_id), maxseconds=maxseconds)\n  File \"/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 403, in wait_for\n    wait_on(func, what, walltime_exceeded)\n  File \"/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/wait.py\", line 38, in wait_on\n    value = function()\n            ^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 399, in \u003clambda\u003e\n    self.wait_for(lambda: self.__job_ready(job_id, history_id), maxseconds=maxseconds)\n                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 749, in __job_ready\n    return self._state_ready(job_id, error_msg=\"Job in error state.\")\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 841, in _state_ready\n    raise Exception(\nException: Job in error state.. tool_id: __DATA_FETCH__, exit_code: None, stderr: .\n", "status": "error", "test_index": 6, "time_seconds": 10.644328117370605, "tool_id": "bcftools_norm", "tool_version": "1.15.1+galaxy4"}
            ]]></error>
        
    </testcase>
    
    <testcase classname="bcftools_norm (Test #8)" name="7" time="31.445247411727905">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            Lines   total/split/realigned/skipped:39/20/3/0
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="bcftools_norm (Test #9)" name="8" time="">
        
            <error type="error" message=""><![CDATA[
                { "status": "skip", "test_index": 8, "tool_id": "bcftools_norm", "tool_version": "1.15.1+galaxy4"}
            ]]></error>
        
    </testcase>
    
    <testcase classname="bcftools_plugin_fill_tags (Test #1)" name="0" time="35.26852488517761">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="bcftools_plugin_fill_tags (Test #2)" name="1" time="30.167759895324707">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="circos (Test #1)" name="0" time="74.34678292274475">
        
            
            <system-out><![CDATA[
            debuggroup summary 0.12s welcome to circos v0.69-8 15 Jun 2019 on Perl 5.032001debuggroup summary 0.12s current working directory /galaxy/server/database/jobs_directory/000/107/workingdebuggroup summary 0.12s command /usr/local/bin/circos -conf circos/conf/circos.conf -noparanoiddebuggroup summary 0.12s loading configuration from file circos/conf/circos.confdebuggroup summary 0.12s found conf file circos/conf/circos.confdebuggroup summary 0.21s debug will appear for these features: output,summarydebuggroup summary 0.21s bitmap output image ./circos.pngdebuggroup summary 0.21s SVG output image ./circos.svgdebuggroup summary 0.21s parsing karyotype and organizing ideogramsdebuggroup summary 0.22s karyotype has 2 chromosomes of total size 56,082debuggroup summary 0.22s applying global and local scalingdebuggroup summary 0.22s allocating image, colors and brushesdebuggroup summary 4.48s drawing 2 ideograms of total size 56,082debuggroup summary 4.48s drawing highlights and ideogramsdebuggroup output 4.48s generating outputdebuggroup output 4.79s created PNG image ./circos.png (40 kb)debuggroup output 4.79s created SVG image ./circos.svg (2 kb)
            ]]></system-out>
            <system-err><![CDATA[
            Use of uninitialized value in subroutine entry at /usr/local/bin/../lib/Circos/Configuration.pm line 781.
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="circos (Test #2)" name="1" time="45.453304290771484">
        
            
            <system-out><![CDATA[
            debuggroup summary 0.12s welcome to circos v0.69-8 15 Jun 2019 on Perl 5.032001debuggroup summary 0.12s current working directory /galaxy/server/database/jobs_directory/000/112/workingdebuggroup summary 0.12s command /usr/local/bin/circos -conf circos/conf/circos.conf -noparanoiddebuggroup summary 0.12s loading configuration from file circos/conf/circos.confdebuggroup summary 0.12s found conf file circos/conf/circos.confdebuggroup summary 0.22s debug will appear for these features: output,summarydebuggroup summary 0.22s bitmap output image ./circos.pngdebuggroup summary 0.22s SVG output image ./circos.svgdebuggroup summary 0.22s parsing karyotype and organizing ideogramsdebuggroup summary 0.22s karyotype has 2 chromosomes of total size 56,082debuggroup summary 0.22s applying global and local scalingdebuggroup summary 0.22s allocating image, colors and brushesdebuggroup summary 4.56s drawing 2 ideograms of total size 56,082debuggroup summary 4.56s drawing highlights and ideogramsdebuggroup summary 4.59s found conf file /usr/local/bin/../etc/tracks/link.confdebuggroup summary 4.59s process track_0 link circos/conf/../data/links-0.txtdebuggroup summary 4.60s drawing link track_0 z 0debuggroup !circoswarning 4.63s  Data point of type [link] [50000-50500] extended past end of ideogram [ctgA 0-50001]. This data point will be [trimmed].debuggroup !circoswarning 4.63s  Data point of type [link] [51000-51500] extended past end of ideogram [ctgA 0-50001]. This data point will be [hidden].debuggroup summary 4.63s found conf file /usr/local/bin/../etc/tracks/histogram.confdebuggroup summary 4.63s found conf file /usr/local/bin/../etc/tracks/scatter.confdebuggroup summary 4.63s found conf file /usr/local/bin/../etc/tracks/tile.confdebuggroup summary 4.63s processing track_0 histogram circos/conf/../data/data-0.txtdebuggroup summary 4.69s processing track_1 scatter circos/conf/../data/data-1.txtdebuggroup summary 4.84s processing track_2 tile circos/conf/../data/data-2.txtdebuggroup summary 4.85s drawing track_0 histogram z 0 data-0.txt orient outdebuggroup summary 4.87s found conf file /usr/local/bin/../etc/tracks/axis.confdebuggroup summary 5.31s drawing track_1 scatter z 0 data-1.txt orient outdebuggroup summary 5.32s found conf file /usr/local/bin/../etc/tracks/axis.confdebuggroup summary 5.45s drawing track_2 tile z 0 data-2.txt orient outdebuggroup summary 5.45s found conf file /usr/local/bin/../etc/tracks/axis.confdebuggroup !circoswarning 5.57s  Data point of type [tile] [3013-6130] extended past end of ideogram [ctgB 0-6079]. This data point will be [trimmed].debuggroup output 5.57s generating outputdebuggroup output 5.95s created PNG image ./circos.png (539 kb)debuggroup output 5.95s created SVG image ./circos.svg (414 kb)
            ]]></system-out>
            <system-err><![CDATA[
            Use of uninitialized value in subroutine entry at /usr/local/bin/../lib/Circos/Configuration.pm line 781.Use of uninitialized value in subroutine entry at /usr/local/bin/../lib/Circos/Configuration.pm line 781.Use of uninitialized value within %Circos::Error::GROUPERROR in uc at /usr/local/bin/../lib/Circos/Error.pm line 354, <F> line 21.Use of uninitialized value within %Circos::Error::GROUPERROR in uc at /usr/local/bin/../lib/Circos/Error.pm line 354, <F> line 21.Use of uninitialized value in subroutine entry at /usr/local/bin/../lib/Circos/Configuration.pm line 781, <F> line 21.Use of uninitialized value in subroutine entry at /usr/local/bin/../lib/Circos/Configuration.pm line 781.Use of uninitialized value in subroutine entry at /usr/local/bin/../lib/Circos/Configuration.pm line 781.Use of uninitialized value in subroutine entry at /usr/local/bin/../lib/Circos/Configuration.pm line 781, <F> line 1214.Use of uninitialized value in subroutine entry at /usr/local/bin/../lib/Circos/Configuration.pm line 781.Use of uninitialized value in subroutine entry at /usr/local/bin/../lib/Circos/Configuration.pm line 781.Use of uninitialized value within %Circos::Error::GROUPERROR in uc at /usr/local/bin/../lib/Circos/Error.pm line 354.
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="deeptools_plot_enrichment (Test #1)" name="0" time="52.0295250415802">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="filter_tabular (Test #1)" name="0" time="72.87642097473145">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="filter_tabular (Test #2)" name="1" time="39.01112961769104">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="filter_tabular (Test #3)" name="2" time="10.149374723434448">
        
            <error type="error" message="Input staging problem: Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 1436, in verify_tool
    stage_data_in_history(
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 175, in stage_data_in_history
    staging_interface.handle_jobs()
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 212, in handle_jobs
    self.galaxy_interactor.wait_for_job(job_response["id"], job_response["history_id"], self.maxseconds)
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 399, in wait_for_job
    self.wait_for(lambda: self.__job_ready(job_id, history_id), maxseconds=maxseconds)
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 403, in wait_for
    wait_on(func, what, walltime_exceeded)
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/wait.py", line 38, in wait_on
    value = function()
            ^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 399, in <lambda>
    self.wait_for(lambda: self.__job_ready(job_id, history_id), maxseconds=maxseconds)
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 749, in __job_ready
    return self._state_ready(job_id, error_msg="Job in error state.")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 841, in _state_ready
    raise Exception(
Exception: Job in error state.. tool_id: __DATA_FETCH__, exit_code: None, stderr: .
"><![CDATA[
                { "execution_problem": "Input staging problem: Traceback (most recent call last):\n  File \"/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 1436, in verify_tool\n    stage_data_in_history(\n  File \"/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 175, in stage_data_in_history\n    staging_interface.handle_jobs()\n  File \"/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 212, in handle_jobs\n    self.galaxy_interactor.wait_for_job(job_response[\"id\"], job_response[\"history_id\"], self.maxseconds)\n  File \"/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 399, in wait_for_job\n    self.wait_for(lambda: self.__job_ready(job_id, history_id), maxseconds=maxseconds)\n  File \"/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 403, in wait_for\n    wait_on(func, what, walltime_exceeded)\n  File \"/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/wait.py\", line 38, in wait_on\n    value = function()\n            ^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 399, in \u003clambda\u003e\n    self.wait_for(lambda: self.__job_ready(job_id, history_id), maxseconds=maxseconds)\n                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 749, in __job_ready\n    return self._state_ready(job_id, error_msg=\"Job in error state.\")\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 841, in _state_ready\n    raise Exception(\nException: Job in error state.. tool_id: __DATA_FETCH__, exit_code: None, stderr: .\n", "status": "error", "test_index": 2, "time_seconds": 10.149374723434448, "tool_id": "filter_tabular", "tool_version": "3.3.1"}
            ]]></error>
        
    </testcase>
    
    <testcase classname="filter_tabular (Test #4)" name="3" time="11.076780080795288">
        
            <error type="error" message="Input staging problem: Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 1436, in verify_tool
    stage_data_in_history(
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 175, in stage_data_in_history
    staging_interface.handle_jobs()
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 212, in handle_jobs
    self.galaxy_interactor.wait_for_job(job_response["id"], job_response["history_id"], self.maxseconds)
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 399, in wait_for_job
    self.wait_for(lambda: self.__job_ready(job_id, history_id), maxseconds=maxseconds)
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 403, in wait_for
    wait_on(func, what, walltime_exceeded)
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/wait.py", line 38, in wait_on
    value = function()
            ^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 399, in <lambda>
    self.wait_for(lambda: self.__job_ready(job_id, history_id), maxseconds=maxseconds)
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 749, in __job_ready
    return self._state_ready(job_id, error_msg="Job in error state.")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 841, in _state_ready
    raise Exception(
Exception: Job in error state.. tool_id: __DATA_FETCH__, exit_code: None, stderr: .
"><![CDATA[
                { "execution_problem": "Input staging problem: Traceback (most recent call last):\n  File \"/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 1436, in verify_tool\n    stage_data_in_history(\n  File \"/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 175, in stage_data_in_history\n    staging_interface.handle_jobs()\n  File \"/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 212, in handle_jobs\n    self.galaxy_interactor.wait_for_job(job_response[\"id\"], job_response[\"history_id\"], self.maxseconds)\n  File \"/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 399, in wait_for_job\n    self.wait_for(lambda: self.__job_ready(job_id, history_id), maxseconds=maxseconds)\n  File \"/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 403, in wait_for\n    wait_on(func, what, walltime_exceeded)\n  File \"/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/wait.py\", line 38, in wait_on\n    value = function()\n            ^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 399, in \u003clambda\u003e\n    self.wait_for(lambda: self.__job_ready(job_id, history_id), maxseconds=maxseconds)\n                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 749, in __job_ready\n    return self._state_ready(job_id, error_msg=\"Job in error state.\")\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 841, in _state_ready\n    raise Exception(\nException: Job in error state.. tool_id: __DATA_FETCH__, exit_code: None, stderr: .\n", "status": "error", "test_index": 3, "time_seconds": 11.076780080795288, "tool_id": "filter_tabular", "tool_version": "3.3.1"}
            ]]></error>
        
    </testcase>
    
    <testcase classname="filter_tabular (Test #5)" name="4" time="28.94436502456665">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="flash (Test #1)" name="0" time="28.72557234764099">
        
            
                <error type="error" message="Tool exit code: None"><![CDATA[
                    { "inputs": {  "generate_histogram": false,  "layout|forward": {   "id": "b7d680e10fa2b4e3",   "src": "hda"  },  "layout|reverse": {   "id": "eafc7c63067206f3",   "src": "hda"  },  "layout|select_layout": "individual",  "save_log": true }, "job": {  "command_line": "flash --threads=${GALAXY_SLOTS:-1} -m 10 -M 65 -x 0.25  \u0027/galaxy/server/database/objects/0/b/5/dataset_0b5cbb8b-cacc-4cb2-b3f3-9d2ef23534c6.dat\u0027 \u0027/galaxy/server/database/objects/5/8/0/dataset_58043119-5061-489f-90c1-44b235e01e0a.dat\u0027 -p 33 --output-suffix=\u0027\u0027 \u003e \u0027/galaxy/server/database/objects/7/f/6/dataset_7f613120-9b93-4d4c-94d6-2000a56038f5.dat\u0027",  "command_version": null,  "copied_from_job_id": null,  "create_time": "2025-09-04T00:57:52.601876",  "dependencies": [],  "exit_code": null,  "external_id": "gxy-d86px",  "galaxy_version": "24.1",  "handler": null,  "history_id": "33bbf33f0e9eba2b",  "id": "bc6ccb8dcbe8dcec",  "inputs": {   "layout|forward": {    "id": "b7d680e10fa2b4e3",    "src": "hda",    "uuid": "0b5cbb8b-cacc-4cb2-b3f3-9d2ef23534c6"   },   "layout|reverse": {    "id": "eafc7c63067206f3",    "src": "hda",    "uuid": "58043119-5061-489f-90c1-44b235e01e0a"   }  },  "job_messages": null,  "job_metrics": [],  "job_runner_name": null,  "job_stderr": null,  "job_stdout": null,  "model_class": "Job",  "output_collections": {},  "outputs": {   "hist": {    "id": "d696602e2780bc5c",    "src": "hda",    "uuid": "817338b5-7022-43b4-acb3-e4aa3691ccfc"   },   "log": {    "id": "cce5cca53d8d21dc",    "src": "hda",    "uuid": "7f613120-9b93-4d4c-94d6-2000a56038f5"   },   "merged_paired_reads": {    "id": "bc6ccb8dcbe8dcec",    "src": "hda",    "uuid": "b3f4cc79-d037-4ee1-b5a6-d88b007aaf92"   },   "unmerged_paired_reads_f": {    "id": "51732cb8072d0c30",    "src": "hda",    "uuid": "16bd1638-05f1-4f74-a215-05659afb36fe"   },   "unmerged_paired_reads_r": {    "id": "6fe389070168ae13",    "src": "hda",    "uuid": "bff368c4-53d7-4e5a-82c4-5039855807da"   }  },  "params": {   "__input_ext": "\"input\"",   "allow_outies": "false",   "chromInfo": "\"/cvmfs/data.galaxyproject.org/managed/len/ucsc/?.len\"",   "dbkey": "\"?\"",   "generate_histogram": "false",   "layout": "{\"__current_case__\": 0, \"forward\": {\"values\": [{\"id\": 9, \"src\": \"hda\"}]}, \"reverse\": {\"values\": [{\"id\": 10, \"src\": \"hda\"}]}, \"select_layout\": \"individual\"}",   "max_mismatch_density": "\"0.25\"",   "max_overlap": "\"65\"",   "min_overlap": "\"10\"",   "save_log": "true"  },  "state": "error",  "stderr": "",  "stdout": "",  "tool_id": "toolshed.g2.bx.psu.edu/repos/iuc/flash/flash/1.2.11.4",  "tool_stderr": "",  "tool_stdout": "",  "update_time": "2025-09-04T00:58:01.541624",  "user_email": "tests@fake.org" }, "output_problems": [  "Job in error state.. tool_id: toolshed.g2.bx.psu.edu/repos/iuc/flash/flash/1.2.11.4, exit_code: None, stderr: .",  "Job in error state.. tool_id: toolshed.g2.bx.psu.edu/repos/iuc/flash/flash/1.2.11.4, exit_code: None, stderr: .",  "Job in error state.. tool_id: toolshed.g2.bx.psu.edu/repos/iuc/flash/flash/1.2.11.4, exit_code: None, stderr: .",  "Job in error state.. tool_id: toolshed.g2.bx.psu.edu/repos/iuc/flash/flash/1.2.11.4, exit_code: None, stderr: .",  "Job in error state.. tool_id: toolshed.g2.bx.psu.edu/repos/iuc/flash/flash/1.2.11.4, exit_code: None, stderr: .",  "Job in error state.. tool_id: toolshed.g2.bx.psu.edu/repos/iuc/flash/flash/1.2.11.4, exit_code: None, stderr: ." ], "status": "failure", "test_index": 0, "time_seconds": 28.72557234764099, "tool_id": "flash", "tool_version": "1.2.11.4"}
                ]]></error>
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="flash (Test #2)" name="1" time="34.30026841163635">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="flash (Test #3)" name="2" time="31.716976165771484">
        
            
            <system-out><![CDATA[
            [FLASH] Starting FLASH v1.2.11[FLASH] Fast Length Adjustment of SHort reads[FLASH]  [FLASH] Input files:[FLASH]     /galaxy/server/database/objects/3/5/b/dataset_35b77e51-4250-49d1-aa07-b243ce72b83a.dat[FLASH]     /galaxy/server/database/objects/3/f/e/dataset_3fece137-ba14-4cd2-839d-2fe06e477f51.dat[FLASH]  [FLASH] Output files:[FLASH]     ./out.extendedFrags.fastq[FLASH]     ./out.notCombined_1.fastq[FLASH]     ./out.notCombined_2.fastq[FLASH]     ./out.hist[FLASH]     ./out.histogram[FLASH]  [FLASH] Parameters:[FLASH]     Min overlap:           10[FLASH]     Max overlap:           65[FLASH]     Max mismatch density:  0.250000[FLASH]     Allow "outie" pairs:   false[FLASH]     Cap mismatch quals:    false[FLASH]     Combiner threads:      1[FLASH]     Input format:          FASTQ, phred_offset=64[FLASH]     Output format:         FASTQ, phred_offset=64[FLASH]  [FLASH] Starting reader and writer threads[FLASH] Starting 1 combiner threads[FLASH] Processed 200 read pairs[FLASH]  [FLASH] Read combination statistics:[FLASH]     Total pairs:      200[FLASH]     Combined pairs:   5[FLASH]     Uncombined pairs: 195[FLASH]     Percent combined: 2.50%[FLASH]  [FLASH] Writing histogram files.[FLASH]  [FLASH] FLASH v1.2.11 complete![FLASH] 0.016 seconds elapsed
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="flash (Test #4)" name="3" time="31.726741552352905">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="flash (Test #5)" name="4" time="31.02009081840515">
        
            
            <system-out><![CDATA[
            [FLASH] Starting FLASH v1.2.11[FLASH] Fast Length Adjustment of SHort reads[FLASH]  [FLASH] Input files:[FLASH]     /galaxy/server/database/objects/7/0/8/dataset_708719bb-d782-4224-a0ed-7b94446aa0fb.dat[FLASH]     /galaxy/server/database/objects/0/0/6/dataset_006c63cc-d61a-44d6-b2ed-5b393a27b81f.dat[FLASH]  [FLASH] Output files:[FLASH]     ./out.extendedFrags.fastq[FLASH]     ./out.notCombined_1.fastq[FLASH]     ./out.notCombined_2.fastq[FLASH]     ./out.hist[FLASH]     ./out.histogram[FLASH]  [FLASH] Parameters:[FLASH]     Min overlap:           10[FLASH]     Max overlap:           65[FLASH]     Max mismatch density:  0.250000[FLASH]     Allow "outie" pairs:   false[FLASH]     Cap mismatch quals:    false[FLASH]     Combiner threads:      1[FLASH]     Input format:          FASTQ, phred_offset=33[FLASH]     Output format:         FASTQ, phred_offset=33, gzip[FLASH]  [FLASH] Starting reader and writer threads[FLASH] Starting 1 combiner threads[FLASH] Processed 500 read pairs[FLASH]  [FLASH] Read combination statistics:[FLASH]     Total pairs:      500[FLASH]     Combined pairs:   18[FLASH]     Uncombined pairs: 482[FLASH]     Percent combined: 3.60%[FLASH]  [FLASH] Writing histogram files.[FLASH]  [FLASH] FLASH v1.2.11 complete![FLASH] 0.031 seconds elapsed
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="ivar_filtervariants (Test #1)" name="0" time="39.34793186187744">
        
            
            <system-out><![CDATA[
            Header format of ./var_0.tsv did not match! Please use files generated using "ivar variants" command.
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="medaka_consensus (Test #1)" name="0" time="69.5660412311554">
        
            
            <system-out><![CDATA[
            [01:12:32 - Predict] Reducing threads to 2, anymore is a waste.[01:12:32 - Predict] It looks like you are running medaka without a GPU and attempted to set a high number of threads. We have scaled this down to an optimal number. If you wish to improve performance please see https://nanoporetech.github.io/medaka/installation.html#improving-parallelism.[01:12:32 - Predict] Setting tensorflow inter/intra-op threads to 2/1.[01:12:32 - Predict] Processing region(s): ref:0-30[01:12:32 - Predict] Using model: /usr/local/lib/python3.8/site-packages/medaka/data/r941_min_high_g360_model.hdf5.[01:12:32 - DataStre] Writing sample registry.[01:12:32 - Predict] Processing 1 long region(s) with batching.[01:12:32 - MdlStore] filepath /usr/local/lib/python3.8/site-packages/medaka/data/r941_min_high_g360_model.hdf52025-09-04 01:12:32.537175: I tensorflow/core/platform/cpu_feature_guard.cc:151] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  SSE4.1 SSE4.2 AVX AVX2 FMATo enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.[01:12:33 - BAMFile] Creating pool of 16 BAM file sets.[01:12:33 - DLoader] Sample-0: started[01:12:33 - Sampler] Initializing sampler for consensus of region ref:0-30.[01:12:33 - DLoader] Sample-1: started[01:12:33 - DLoader] Sample-1: finished[01:12:33 - Feature] Pileup counts do not span requested region, requested ref:0-30, received 0-9.[01:12:33 - Feature] Processed ref:0.0-9.0 (median depth 1.0)[01:12:33 - Feature] Pileup counts do not span requested region, requested ref:0-30, received 14-29.[01:12:33 - Feature] Processed ref:14.0-29.0 (median depth 1.0)[01:12:33 - Sampler] Took 0.02s to make features.[01:12:33 - Sampler] Region ref:0.0-9.0 (10 positions) is smaller than inference chunk length 10000, quarantining.[01:12:33 - Sampler] Region ref:14.0-29.0 (16 positions) is smaller than inference chunk length 10000, quarantining.[01:12:33 - DLoader] Sample-0: finished[01:12:33 - DLoader] Initialised. Batch size:100. Workers:2. Cache:800[01:12:33 - PWorker] Running inference for 0.0M draft bases.[01:12:33 - DLoader] Batches ready: 0. Samples ready: 0 (0.0 batches)[01:12:33 - DataStre] Writing sample registry.[01:12:33 - PWorker] Processed 0 batches[01:12:33 - PWorker] All done, 2 remainder regions.[01:12:33 - Predict] Processing 2 short region(s).[01:12:33 - MdlStore] filepath /usr/local/lib/python3.8/site-packages/medaka/data/r941_min_high_g360_model.hdf5[01:12:33 - DLoader] Sample-0: started[01:12:33 - Sampler] Initializing sampler for consensus of region ref:0-10.[01:12:33 - DLoader] Sample-1: started[01:12:33 - Sampler] Initializing sampler for consensus of region ref:14-30.[01:12:33 - DLoader] Initialised. Batch size:1. Workers:2. Cache:8[01:12:33 - PWorker] Running inference for 0.0M draft bases.[01:12:33 - Feature] Processed ref:0.0-9.0 (median depth 1.0)[01:12:33 - Sampler] Took 0.03s to make features.[01:12:33 - Sampler] Pileup for ref:0.0-9.0 is of width 10[01:12:33 - DLoader] Sample-0: finished[01:12:33 - Feature] Processed ref:14.0-29.0 (median depth 1.0)[01:12:33 - Sampler] Took 0.04s to make features.[01:12:33 - Sampler] Pileup for ref:14.0-29.0 is of width 16[01:12:33 - DLoader] Batches ready: 0. Samples ready: 0 (0.0 batches)[01:12:33 - DLoader] Sample-1: finished[01:12:33 - DataStre] Loaded sample register.[01:12:33 - DataStre] Adding ref:0.0-9.0 to sample registry[01:12:33 - DLoader] Batches ready: 1. Samples ready: 0 (0.0 batches)[01:12:33 - DataStre] Adding ref:14.0-29.0 to sample registry[01:12:33 - DLoader] Batches ready: 0. Samples ready: 0 (0.0 batches)[01:12:33 - DataStre] Writing sample registry.[01:12:33 - PWorker] Processed 2 batches[01:12:33 - PWorker] All done, 0 remainder regions.[01:12:33 - Predict] Finished processing all regions.[01:12:33 - BAMFile] Closing BAM file set.[01:12:33 - BAMFile] Closing BAM file set.[01:12:33 - BAMFile] Closing BAM file set.[01:12:33 - BAMFile] Closing BAM file set.[01:12:33 - BAMFile] Closing BAM file set.[01:12:33 - BAMFile] Closing BAM file set.[01:12:33 - BAMFile] Closing BAM file set.[01:12:33 - BAMFile] Closing BAM file set.[01:12:33 - BAMFile] Closing BAM file set.[01:12:33 - BAMFile] Closing BAM file set.[01:12:33 - BAMFile] Closing BAM file set.[01:12:33 - BAMFile] Closing BAM file set.[01:12:33 - BAMFile] Closing BAM file set.[01:12:33 - BAMFile] Closing BAM file set.[01:12:33 - BAMFile] Closing BAM file set.[01:12:33 - BAMFile] Closing BAM file set.
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="medaka_consensus (Test #2)" name="1" time="39.15649080276489">
        
            
            <system-out><![CDATA[
            [01:13:18 - Predict] Reducing threads to 2, anymore is a waste.[01:13:18 - Predict] It looks like you are running medaka without a GPU and attempted to set a high number of threads. We have scaled this down to an optimal number. If you wish to improve performance please see https://nanoporetech.github.io/medaka/installation.html#improving-parallelism.[01:13:18 - Predict] Setting tensorflow inter/intra-op threads to 2/1.[01:13:18 - Predict] Processing region(s): ref:0-30[01:13:18 - Predict] Using model: /usr/local/lib/python3.8/site-packages/medaka/data/r941_min_fast_g303_model.hdf5.[01:13:18 - DataStre] Writing sample registry.[01:13:18 - Predict] Processing 1 long region(s) with batching.[01:13:18 - MdlStore] filepath /usr/local/lib/python3.8/site-packages/medaka/data/r941_min_fast_g303_model.hdf52025-09-04 01:13:18.224280: I tensorflow/core/platform/cpu_feature_guard.cc:151] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  SSE4.1 SSE4.2 AVX AVX2 FMATo enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.[01:13:18 - BAMFile] Creating pool of 16 BAM file sets.[01:13:18 - DLoader] Sample-0: started[01:13:18 - Sampler] Initializing sampler for consensus of region ref:0-30.[01:13:18 - DLoader] Sample-1: started[01:13:18 - DLoader] Sample-1: finished[01:13:18 - DLoader] Initialised. Batch size:99. Workers:2. Cache:792[01:13:18 - PWorker] Running inference for 0.0M draft bases.[01:13:18 - Feature] Pileup counts do not span requested region, requested ref:0-30, received 0-9.[01:13:18 - Feature] Processed ref:0.0-9.0 (median depth 1.0)[01:13:18 - Feature] Pileup counts do not span requested region, requested ref:0-30, received 14-29.[01:13:18 - Feature] Processed ref:14.0-29.0 (median depth 1.0)[01:13:18 - Sampler] Took 0.09s to make features.[01:13:18 - Sampler] Region ref:0.0-9.0 (10 positions) is smaller than inference chunk length 9999, quarantining.[01:13:18 - Sampler] Region ref:14.0-29.0 (16 positions) is smaller than inference chunk length 9999, quarantining.[01:13:18 - DLoader] Sample-0: finished[01:13:18 - DLoader] Batches ready: 0. Samples ready: 0 (0.0 batches)[01:13:18 - DataStre] Writing sample registry.[01:13:18 - PWorker] Processed 0 batches[01:13:18 - PWorker] All done, 2 remainder regions.[01:13:18 - Predict] Processing 2 short region(s).[01:13:18 - MdlStore] filepath /usr/local/lib/python3.8/site-packages/medaka/data/r941_min_fast_g303_model.hdf5[01:13:19 - DLoader] Sample-0: started[01:13:19 - Sampler] Initializing sampler for consensus of region ref:0-10.[01:13:19 - DLoader] Sample-1: started[01:13:19 - Sampler] Initializing sampler for consensus of region ref:14-30.[01:13:19 - DLoader] Initialised. Batch size:1. Workers:2. Cache:8[01:13:19 - PWorker] Running inference for 0.0M draft bases.[01:13:19 - Feature] Processed ref:0.0-9.0 (median depth 1.0)[01:13:19 - Sampler] Took 0.03s to make features.[01:13:19 - Sampler] Pileup for ref:0.0-9.0 is of width 10[01:13:19 - DLoader] Sample-0: finished[01:13:19 - Feature] Processed ref:14.0-29.0 (median depth 1.0)[01:13:19 - Sampler] Took 0.03s to make features.[01:13:19 - Sampler] Pileup for ref:14.0-29.0 is of width 16[01:13:19 - DLoader] Sample-1: finished[01:13:19 - DLoader] Batches ready: 0. Samples ready: 1 (1.0 batches)[01:13:19 - DataStre] Loaded sample register.[01:13:19 - DataStre] Adding ref:0.0-9.0 to sample registry[01:13:19 - DLoader] Batches ready: 1. Samples ready: 0 (0.0 batches)[01:13:19 - DataStre] Adding ref:14.0-29.0 to sample registry[01:13:19 - DLoader] Batches ready: 0. Samples ready: 0 (0.0 batches)[01:13:19 - DataStre] Writing sample registry.[01:13:19 - PWorker] Processed 2 batches[01:13:19 - PWorker] All done, 0 remainder regions.[01:13:19 - Predict] Finished processing all regions.[01:13:19 - Predict] Validating and finalising output data.[01:13:19 - DataStre] Writing sample registry.[01:13:19 - DataStre] Created missing sample register.[01:13:19 - BAMFile] Closing BAM file set.[01:13:19 - BAMFile] Closing BAM file set.[01:13:19 - BAMFile] Closing BAM file set.[01:13:19 - BAMFile] Closing BAM file set.[01:13:19 - BAMFile] Closing BAM file set.[01:13:19 - BAMFile] Closing BAM file set.[01:13:19 - BAMFile] Closing BAM file set.[01:13:19 - BAMFile] Closing BAM file set.[01:13:19 - BAMFile] Closing BAM file set.[01:13:19 - BAMFile] Closing BAM file set.[01:13:19 - BAMFile] Closing BAM file set.[01:13:19 - BAMFile] Closing BAM file set.[01:13:19 - BAMFile] Closing BAM file set.[01:13:19 - BAMFile] Closing BAM file set.[01:13:19 - BAMFile] Closing BAM file set.[01:13:19 - BAMFile] Closing BAM file set.
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="poretools_nucdist (Test #1)" name="0" time="56.817195415496826">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="poretools_nucdist (Test #2)" name="1" time="31.76489520072937">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="qualimap_multi_bamqc (Test #1)" name="0" time="104.12918496131897">
        
            
            <system-out><![CDATA[
            Java memory size is set to 1200MLaunching application...detected environment java options -Djava.awt.headless=true -Xmx4080mQualiMap v.2.3Built on 2023-05-19 16:57Selected tool: multi-bamqcRunning multi-sample BAM QCChecking input pathsLoading sample dataCreating chartsPreparing result reportWriting HTML report...HTML report created successfullyFinished
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="racon (Test #1)" name="0" time="43.78150677680969">
        
            
                <error type="error" message="Tool exit code: 132"><![CDATA[
                    { "inputs": {  "corrected_reads": {   "id": "154f7cc7a80a8144",   "src": "hda"  },  "error_threshold": "0.2",  "fragment_correction": true,  "include_unpolished": true,  "overlaps": {   "id": "f4df901453f498e4",   "src": "hda"  },  "reads": {   "id": "8955e5b88e7b3e59",   "src": "hda"  },  "window_length": "800" }, "job": {  "command_line": "ln -s \u0027/galaxy/server/database/objects/e/e/2/dataset_ee22c4a2-025b-4eb7-b004-19e5ee051771.dat\u0027 reads.fasta \u0026\u0026 ln -s \u0027/galaxy/server/database/objects/f/8/7/dataset_f87fe11a-27d1-4dc1-83c6-5cc6ac8718b1.dat\u0027 overlaps.sam \u0026\u0026  ln -s \u0027/galaxy/server/database/objects/5/1/2/dataset_5123111c-d6eb-4b98-894a-cc62365b2bc6.dat\u0027 corrected_reads.fasta \u0026\u0026  racon reads.fasta overlaps.sam corrected_reads.fasta -t ${GALAXY_SLOTS:-4} -u -f -w 800 -q 10.0 -e 0.2  -m 3 -x -5 -g -4 \u003e racon_polished_consensus.fa",  "command_version": "1.5.0",  "copied_from_job_id": null,  "create_time": "2025-09-04T01:16:19.160274",  "dependencies": [],  "exit_code": 132,  "external_id": "gxy-s4b2c",  "galaxy_version": "24.1",  "handler": null,  "history_id": "33bbf33f0e9eba2b",  "id": "69cc85c96ee4c235",  "inputs": {   "corrected_reads": {    "id": "154f7cc7a80a8144",    "src": "hda",    "uuid": "5123111c-d6eb-4b98-894a-cc62365b2bc6"   },   "overlaps": {    "id": "f4df901453f498e4",    "src": "hda",    "uuid": "f87fe11a-27d1-4dc1-83c6-5cc6ac8718b1"   },   "reads": {    "id": "8955e5b88e7b3e59",    "src": "hda",    "uuid": "ee22c4a2-025b-4eb7-b004-19e5ee051771"   }  },  "job_messages": [   {    "code_desc": "",    "desc": "Fatal error: Exit code 132 ()",    "error_level": 3,    "exit_code": 132,    "type": "exit_code"   }  ],  "job_metrics": [   {    "name": "runtime_seconds",    "plugin": "core",    "raw_value": "0E-7",    "title": "Job Runtime (Wall Clock)",    "value": "0 seconds"   },   {    "name": "end_epoch",    "plugin": "core",    "raw_value": "1756948588.0000000",    "title": "Job End Time",    "value": "2025-09-04 01:16:28"   },   {    "name": "start_epoch",    "plugin": "core",    "raw_value": "1756948588.0000000",    "title": "Job Start Time",    "value": "2025-09-04 01:16:28"   },   {    "name": "galaxy_memory_mb",    "plugin": "core",    "raw_value": "38654.0000000",    "title": "Memory Allocated (MB)",    "value": "38654"   },   {    "name": "galaxy_slots",    "plugin": "core",    "raw_value": "8.0000000",    "title": "Cores Allocated",    "value": "8"   }  ],  "job_runner_name": null,  "job_stderr": "",  "job_stdout": "",  "model_class": "Job",  "output_collections": {},  "outputs": {   "consensus": {    "id": "cda33c3c535129c8",    "src": "hda",    "uuid": "e9afe56d-74f3-4d0c-864d-cb9830c662fc"   }  },  "params": {   "__input_ext": "\"input\"",   "chromInfo": "\"/cvmfs/data.galaxyproject.org/managed/len/ucsc/?.len\"",   "dbkey": "\"?\"",   "error_threshold": "\"0.2\"",   "fragment_correction": "true",   "gap": "\"-4\"",   "include_unpolished": "true",   "match": "\"3\"",   "mismatch": "\"-5\"",   "no_trimming": "false",   "quality_threshold": "\"10.0\"",   "window_length": "\"800\""  },  "state": "error",  "stderr": "/galaxy/server/database/jobs_directory/000/94/tool_script.sh: line 24:    27 Illegal instruction     (core dumped) racon reads.fasta overlaps.sam corrected_reads.fasta -t ${GALAXY_SLOTS:-4} -u -f -w 800 -q 10.0 -e 0.2 -m 3 -x -5 -g -4 \u003e racon_polished_consensus.fa\n",  "stdout": "",  "tool_id": "toolshed.g2.bx.psu.edu/repos/bgruening/racon/racon/1.5.0+galaxy1",  "tool_stderr": "/galaxy/server/database/jobs_directory/000/94/tool_script.sh: line 24:    27 Illegal instruction     (core dumped) racon reads.fasta overlaps.sam corrected_reads.fasta -t ${GALAXY_SLOTS:-4} -u -f -w 800 -q 10.0 -e 0.2 -m 3 -x -5 -g -4 \u003e racon_polished_consensus.fa\n",  "tool_stdout": "",  "update_time": "2025-09-04T01:16:39.797553",  "user_email": "tests@fake.org" }, "output_problems": [  "Job in error state.. tool_id: toolshed.g2.bx.psu.edu/repos/bgruening/racon/racon/1.5.0+galaxy1, exit_code: 132, stderr: /galaxy/server/database/jobs_directory/000/94/tool_script.sh: line 24:    27 Illegal instruction     (core dumped) racon reads.fasta overlaps.sam corrected_reads.fasta -t ${GALAXY_SLOTS:-4} -u -f -w 800 -q 10.0 -e 0.2 -m 3 -x -5 -g -4 \u003e racon_polished_consensus.fa\n.",  "Job in error state.. tool_id: toolshed.g2.bx.psu.edu/repos/bgruening/racon/racon/1.5.0+galaxy1, exit_code: 132, stderr: /galaxy/server/database/jobs_directory/000/94/tool_script.sh: line 24:    27 Illegal instruction     (core dumped) racon reads.fasta overlaps.sam corrected_reads.fasta -t ${GALAXY_SLOTS:-4} -u -f -w 800 -q 10.0 -e 0.2 -m 3 -x -5 -g -4 \u003e racon_polished_consensus.fa\n." ], "status": "failure", "test_index": 0, "time_seconds": 43.78150677680969, "tool_id": "racon", "tool_version": "1.5.0+galaxy1"}
                ]]></error>
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            /galaxy/server/database/jobs_directory/000/94/tool_script.sh: line 24:    27 Illegal instruction     (core dumped) racon reads.fasta overlaps.sam corrected_reads.fasta -t ${GALAXY_SLOTS:-4} -u -f -w 800 -q 10.0 -e 0.2 -m 3 -x -5 -g -4 > racon_polished_consensus.fa
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="racon (Test #2)" name="1" time="35.3205680847168">
        
            
                <error type="error" message="Tool exit code: 132"><![CDATA[
                    { "inputs": {  "corrected_reads": {   "id": "f3f3edb85628c1d2",   "src": "hda"  },  "overlaps": {   "id": "4d771f6dd8ffa11f",   "src": "hda"  },  "reads": {   "id": "1cfbfac50985e6c3",   "src": "hda"  } }, "job": {  "command_line": "ln -s \u0027/galaxy/server/database/objects/7/e/6/dataset_7e6ab4fb-ae5f-4730-adbc-e0790cdc55e3.dat\u0027 reads.fasta \u0026\u0026 ln -s \u0027/galaxy/server/database/objects/5/4/8/dataset_548bd531-cd0b-4354-b337-36f3921a2ba7.dat\u0027 overlaps.paf \u0026\u0026  ln -s \u0027/galaxy/server/database/objects/6/e/f/dataset_6ef44168-aebb-40d6-a0b2-35c58e912e82.dat\u0027 corrected_reads.fasta \u0026\u0026  racon reads.fasta overlaps.paf corrected_reads.fasta -t ${GALAXY_SLOTS:-4}   -w 500 -q 10.0 -e 0.3  -m 3 -x -5 -g -4 \u003e racon_polished_consensus.fa",  "command_version": "1.5.0",  "copied_from_job_id": null,  "create_time": "2025-09-04T01:17:03.052258",  "dependencies": [],  "exit_code": 132,  "external_id": "gxy-dpx46",  "galaxy_version": "24.1",  "handler": null,  "history_id": "33bbf33f0e9eba2b",  "id": "4571f79f7880d66d",  "inputs": {   "corrected_reads": {    "id": "f3f3edb85628c1d2",    "src": "hda",    "uuid": "6ef44168-aebb-40d6-a0b2-35c58e912e82"   },   "overlaps": {    "id": "4d771f6dd8ffa11f",    "src": "hda",    "uuid": "548bd531-cd0b-4354-b337-36f3921a2ba7"   },   "reads": {    "id": "1cfbfac50985e6c3",    "src": "hda",    "uuid": "7e6ab4fb-ae5f-4730-adbc-e0790cdc55e3"   }  },  "job_messages": [   {    "code_desc": "",    "desc": "Fatal error: Exit code 132 ()",    "error_level": 3,    "exit_code": 132,    "type": "exit_code"   }  ],  "job_metrics": [   {    "name": "runtime_seconds",    "plugin": "core",    "raw_value": "0E-7",    "title": "Job Runtime (Wall Clock)",    "value": "0 seconds"   },   {    "name": "end_epoch",    "plugin": "core",    "raw_value": "1756948624.0000000",    "title": "Job End Time",    "value": "2025-09-04 01:17:04"   },   {    "name": "start_epoch",    "plugin": "core",    "raw_value": "1756948624.0000000",    "title": "Job Start Time",    "value": "2025-09-04 01:17:04"   },   {    "name": "galaxy_memory_mb",    "plugin": "core",    "raw_value": "38654.0000000",    "title": "Memory Allocated (MB)",    "value": "38654"   },   {    "name": "galaxy_slots",    "plugin": "core",    "raw_value": "8.0000000",    "title": "Cores Allocated",    "value": "8"   }  ],  "job_runner_name": null,  "job_stderr": "",  "job_stdout": "",  "model_class": "Job",  "output_collections": {},  "outputs": {   "consensus": {    "id": "b08f73ee4532d39b",    "src": "hda",    "uuid": "09665654-ba80-419a-a54b-64f4afadac9c"   }  },  "params": {   "__input_ext": "\"input\"",   "chromInfo": "\"/cvmfs/data.galaxyproject.org/managed/len/ucsc/?.len\"",   "dbkey": "\"?\"",   "error_threshold": "\"0.3\"",   "fragment_correction": "false",   "gap": "\"-4\"",   "include_unpolished": "false",   "match": "\"3\"",   "mismatch": "\"-5\"",   "no_trimming": "false",   "quality_threshold": "\"10.0\"",   "window_length": "\"500\""  },  "state": "error",  "stderr": "/galaxy/server/database/jobs_directory/000/98/tool_script.sh: line 24:    27 Illegal instruction     (core dumped) racon reads.fasta overlaps.paf corrected_reads.fasta -t ${GALAXY_SLOTS:-4} -w 500 -q 10.0 -e 0.3 -m 3 -x -5 -g -4 \u003e racon_polished_consensus.fa\n",  "stdout": "",  "tool_id": "toolshed.g2.bx.psu.edu/repos/bgruening/racon/racon/1.5.0+galaxy1",  "tool_stderr": "/galaxy/server/database/jobs_directory/000/98/tool_script.sh: line 24:    27 Illegal instruction     (core dumped) racon reads.fasta overlaps.paf corrected_reads.fasta -t ${GALAXY_SLOTS:-4} -w 500 -q 10.0 -e 0.3 -m 3 -x -5 -g -4 \u003e racon_polished_consensus.fa\n",  "tool_stdout": "",  "update_time": "2025-09-04T01:17:15.073358",  "user_email": "tests@fake.org" }, "output_problems": [  "Job in error state.. tool_id: toolshed.g2.bx.psu.edu/repos/bgruening/racon/racon/1.5.0+galaxy1, exit_code: 132, stderr: /galaxy/server/database/jobs_directory/000/98/tool_script.sh: line 24:    27 Illegal instruction     (core dumped) racon reads.fasta overlaps.paf corrected_reads.fasta -t ${GALAXY_SLOTS:-4} -w 500 -q 10.0 -e 0.3 -m 3 -x -5 -g -4 \u003e racon_polished_consensus.fa\n.",  "Job in error state.. tool_id: toolshed.g2.bx.psu.edu/repos/bgruening/racon/racon/1.5.0+galaxy1, exit_code: 132, stderr: /galaxy/server/database/jobs_directory/000/98/tool_script.sh: line 24:    27 Illegal instruction     (core dumped) racon reads.fasta overlaps.paf corrected_reads.fasta -t ${GALAXY_SLOTS:-4} -w 500 -q 10.0 -e 0.3 -m 3 -x -5 -g -4 \u003e racon_polished_consensus.fa\n." ], "status": "failure", "test_index": 1, "time_seconds": 35.3205680847168, "tool_id": "racon", "tool_version": "1.5.0+galaxy1"}
                ]]></error>
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            /galaxy/server/database/jobs_directory/000/98/tool_script.sh: line 24:    27 Illegal instruction     (core dumped) racon reads.fasta overlaps.paf corrected_reads.fasta -t ${GALAXY_SLOTS:-4} -w 500 -q 10.0 -e 0.3 -m 3 -x -5 -g -4 > racon_polished_consensus.fa
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="racon (Test #3)" name="2" time="34.722676277160645">
        
            
                <error type="error" message="Tool exit code: 132"><![CDATA[
                    { "inputs": {  "corrected_reads": {   "id": "b48701d6643e26d9",   "src": "hda"  },  "fragment_correction": false,  "gap": "-3",  "include_unpolished": false,  "match": "7",  "mismatch": "-4",  "no_trimming": true,  "overlaps": {   "id": "7226d35bc0f0ce1f",   "src": "hda"  },  "quality_threshold": "9.0",  "reads": {   "id": "266d056c16786554",   "src": "hda"  } }, "job": {  "command_line": "ln -s \u0027/galaxy/server/database/objects/2/0/9/dataset_209106bb-008e-4747-9352-cc1cbcd772d1.dat\u0027 reads.fasta \u0026\u0026 ln -s \u0027/galaxy/server/database/objects/d/d/0/dataset_dd0af557-7f30-4993-b071-c262ea71a04c.dat\u0027 overlaps.sam \u0026\u0026  ln -s \u0027/galaxy/server/database/objects/d/4/2/dataset_d42ac5b6-8501-44ea-bcbb-f002543fbfbc.dat\u0027 corrected_reads.fasta \u0026\u0026  racon reads.fasta overlaps.sam corrected_reads.fasta -t ${GALAXY_SLOTS:-4}   -w 500 -q 9.0 -e 0.3 --no-trimming -m 7 -x -4 -g -3 \u003e racon_polished_consensus.fa",  "command_version": "1.5.0",  "copied_from_job_id": null,  "create_time": "2025-09-04T01:17:38.206472",  "dependencies": [],  "exit_code": 132,  "external_id": "gxy-x9jnq",  "galaxy_version": "24.1",  "handler": null,  "history_id": "33bbf33f0e9eba2b",  "id": "f322cd5a2100700e",  "inputs": {   "corrected_reads": {    "id": "b48701d6643e26d9",    "src": "hda",    "uuid": "d42ac5b6-8501-44ea-bcbb-f002543fbfbc"   },   "overlaps": {    "id": "7226d35bc0f0ce1f",    "src": "hda",    "uuid": "dd0af557-7f30-4993-b071-c262ea71a04c"   },   "reads": {    "id": "266d056c16786554",    "src": "hda",    "uuid": "209106bb-008e-4747-9352-cc1cbcd772d1"   }  },  "job_messages": [   {    "code_desc": "",    "desc": "Fatal error: Exit code 132 ()",    "error_level": 3,    "exit_code": 132,    "type": "exit_code"   }  ],  "job_metrics": [   {    "name": "runtime_seconds",    "plugin": "core",    "raw_value": "1.0000000",    "title": "Job Runtime (Wall Clock)",    "value": "1 second"   },   {    "name": "end_epoch",    "plugin": "core",    "raw_value": "1756948660.0000000",    "title": "Job End Time",    "value": "2025-09-04 01:17:40"   },   {    "name": "start_epoch",    "plugin": "core",    "raw_value": "1756948659.0000000",    "title": "Job Start Time",    "value": "2025-09-04 01:17:39"   },   {    "name": "galaxy_memory_mb",    "plugin": "core",    "raw_value": "38654.0000000",    "title": "Memory Allocated (MB)",    "value": "38654"   },   {    "name": "galaxy_slots",    "plugin": "core",    "raw_value": "8.0000000",    "title": "Cores Allocated",    "value": "8"   }  ],  "job_runner_name": null,  "job_stderr": "",  "job_stdout": "",  "model_class": "Job",  "output_collections": {},  "outputs": {   "consensus": {    "id": "e010483be6bc8e81",    "src": "hda",    "uuid": "405858ba-ce2a-47e0-8b09-5a4d98f6e9e3"   }  },  "params": {   "__input_ext": "\"input\"",   "chromInfo": "\"/cvmfs/data.galaxyproject.org/managed/len/ucsc/?.len\"",   "dbkey": "\"?\"",   "error_threshold": "\"0.3\"",   "fragment_correction": "false",   "gap": "\"-3\"",   "include_unpolished": "false",   "match": "\"7\"",   "mismatch": "\"-4\"",   "no_trimming": "true",   "quality_threshold": "\"9.0\"",   "window_length": "\"500\""  },  "state": "error",  "stderr": "/galaxy/server/database/jobs_directory/000/102/tool_script.sh: line 24:    27 Illegal instruction     (core dumped) racon reads.fasta overlaps.sam corrected_reads.fasta -t ${GALAXY_SLOTS:-4} -w 500 -q 9.0 -e 0.3 --no-trimming -m 7 -x -4 -g -3 \u003e racon_polished_consensus.fa\n",  "stdout": "",  "tool_id": "toolshed.g2.bx.psu.edu/repos/bgruening/racon/racon/1.5.0+galaxy1",  "tool_stderr": "/galaxy/server/database/jobs_directory/000/102/tool_script.sh: line 24:    27 Illegal instruction     (core dumped) racon reads.fasta overlaps.sam corrected_reads.fasta -t ${GALAXY_SLOTS:-4} -w 500 -q 9.0 -e 0.3 --no-trimming -m 7 -x -4 -g -3 \u003e racon_polished_consensus.fa\n",  "tool_stdout": "",  "update_time": "2025-09-04T01:17:50.084636",  "user_email": "tests@fake.org" }, "output_problems": [  "Job in error state.. tool_id: toolshed.g2.bx.psu.edu/repos/bgruening/racon/racon/1.5.0+galaxy1, exit_code: 132, stderr: /galaxy/server/database/jobs_directory/000/102/tool_script.sh: line 24:    27 Illegal instruction     (core dumped) racon reads.fasta overlaps.sam corrected_reads.fasta -t ${GALAXY_SLOTS:-4} -w 500 -q 9.0 -e 0.3 --no-trimming -m 7 -x -4 -g -3 \u003e racon_polished_consensus.fa\n.",  "Job in error state.. tool_id: toolshed.g2.bx.psu.edu/repos/bgruening/racon/racon/1.5.0+galaxy1, exit_code: 132, stderr: /galaxy/server/database/jobs_directory/000/102/tool_script.sh: line 24:    27 Illegal instruction     (core dumped) racon reads.fasta overlaps.sam corrected_reads.fasta -t ${GALAXY_SLOTS:-4} -w 500 -q 9.0 -e 0.3 --no-trimming -m 7 -x -4 -g -3 \u003e racon_polished_consensus.fa\n." ], "status": "failure", "test_index": 2, "time_seconds": 34.722676277160645, "tool_id": "racon", "tool_version": "1.5.0+galaxy1"}
                ]]></error>
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            /galaxy/server/database/jobs_directory/000/102/tool_script.sh: line 24:    27 Illegal instruction     (core dumped) racon reads.fasta overlaps.sam corrected_reads.fasta -t ${GALAXY_SLOTS:-4} -w 500 -q 9.0 -e 0.3 --no-trimming -m 7 -x -4 -g -3 > racon_polished_consensus.fa
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="samtools_sort (Test #1)" name="0" time="32.38552808761597">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="samtools_sort (Test #2)" name="1" time="31.357605695724487">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="samtools_sort (Test #3)" name="2" time="30.353684902191162">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="samtools_sort (Test #4)" name="3" time="29.830345153808594">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="samtools_sort (Test #5)" name="4" time="29.21649146080017">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="samtools_sort (Test #6)" name="5" time="31.103524208068848">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="samtools_sort (Test #7)" name="6" time="28.396127939224243">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="samtools_sort (Test #8)" name="7" time="28.107128381729126">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="samtools_sort (Test #9)" name="8" time="30.593274116516113">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            
            ]]></system-err>
        
    </testcase>
    
    <testcase classname="vcfallelicprimitives (Test #1)" name="0" time="10.836370706558228">
        
            <error type="error" message="Input staging problem: Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 1436, in verify_tool
    stage_data_in_history(
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 175, in stage_data_in_history
    staging_interface.handle_jobs()
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 212, in handle_jobs
    self.galaxy_interactor.wait_for_job(job_response["id"], job_response["history_id"], self.maxseconds)
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 399, in wait_for_job
    self.wait_for(lambda: self.__job_ready(job_id, history_id), maxseconds=maxseconds)
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 403, in wait_for
    wait_on(func, what, walltime_exceeded)
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/wait.py", line 38, in wait_on
    value = function()
            ^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 399, in <lambda>
    self.wait_for(lambda: self.__job_ready(job_id, history_id), maxseconds=maxseconds)
                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 749, in __job_ready
    return self._state_ready(job_id, error_msg="Job in error state.")
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py", line 841, in _state_ready
    raise Exception(
Exception: Job in error state.. tool_id: __DATA_FETCH__, exit_code: None, stderr: .
"><![CDATA[
                { "execution_problem": "Input staging problem: Traceback (most recent call last):\n  File \"/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 1436, in verify_tool\n    stage_data_in_history(\n  File \"/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 175, in stage_data_in_history\n    staging_interface.handle_jobs()\n  File \"/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 212, in handle_jobs\n    self.galaxy_interactor.wait_for_job(job_response[\"id\"], job_response[\"history_id\"], self.maxseconds)\n  File \"/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 399, in wait_for_job\n    self.wait_for(lambda: self.__job_ready(job_id, history_id), maxseconds=maxseconds)\n  File \"/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 403, in wait_for\n    wait_on(func, what, walltime_exceeded)\n  File \"/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/wait.py\", line 38, in wait_on\n    value = function()\n            ^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 399, in \u003clambda\u003e\n    self.wait_for(lambda: self.__job_ready(job_id, history_id), maxseconds=maxseconds)\n                          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 749, in __job_ready\n    return self._state_ready(job_id, error_msg=\"Job in error state.\")\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.12.11/x64/lib/python3.12/site-packages/galaxy/tool_util/verify/interactor.py\", line 841, in _state_ready\n    raise Exception(\nException: Job in error state.. tool_id: __DATA_FETCH__, exit_code: None, stderr: .\n", "status": "error", "test_index": 0, "time_seconds": 10.836370706558228, "tool_id": "vcfallelicprimitives", "tool_version": "1.0.0_rc3+galaxy0"}
            ]]></error>
        
    </testcase>
    
    <testcase classname="vcfprimers (Test #1)" name="0" time="37.50237822532654">
        
            
            <system-out><![CDATA[
            
            ]]></system-out>
            <system-err><![CDATA[
            index file localref.fa.fai not found, generating...
            ]]></system-err>
        
    </testcase>
    
</testsuite>